### PR TITLE
Added and modified synonym file

### PR DESCRIPTION
**Pull request by sunilk747**

### CODE REVIEW

File: tests/synonyms/license_synonyms.json [hunk 1]
```diff
@@ -211,7 +209,7 @@
     "Modified BSD License": "bsd-new", 
     "Python style": "unknown", 
     "Common Public License Version 1.0": "cpl 1.0",
-    "MPL 2.0 or EPL 1.0": "unknown", 
+    "MPL 2.0 or EPL 1.0": "mpl 2.0",
```
Comment (miteshvp) [inline tests/synonyms/license_synonyms.json]: Just a random comment, but I feel that we should have lowercase letters for everything here and should be handled via program. In this way we can avoid something like
MPL 2.0 or EPL 1.0 -> mpl 2.0
mpl 2.0 or epl 1.0 -> mpl 2.0
Mpl 2.0 or Epl 1.0 ->  mpl 2.0

Comment (sara-02) [reply tests/synonyms/license_synonyms.json]: @miteshvp Yes, we have had this discussion before, we have to modify this file to all things lower case.

Comment (sunilk747) [reply tests/synonyms/license_synonyms.json]: Yes. I had the same discussion with sara yesterday. If we convert all the input licenses names into lowercase before passing it to our platform that way we can reduce the list of synonyms.

Comment (sara-02) [reply tests/synonyms/license_synonyms.json]: @miteshvp https://github.com/fabric8-analytics/fabric8-analytics-license-analysis/issues/52 we can track it here. :)

**Pull request merged**